### ISSUE TITLE
How can we show play button in YCP?

### ISSUE THREAD

**Issue by mdtaher** [bachors/jQuery-Youtube-Channels-Playlist#8]
Can we show play button using ycp.js,

**Comment by bachors** [comment]
@mdtaher featuring the play button when the image hover ?

**Comment by Creativenauts** [comment]
I too would like to know if it's possible to show the red youtube play button on top of the video.

**Comment by mdtaher** [comment]
@bachors , Yes on "hover" and when video is on "pause" want to show "Youtube" play button. I wanted to implement toggle functionality of youtube play button.
